### PR TITLE
Add force option

### PR DESCRIPTION
**Pull request by rrequero**
I need use force option in the grunt task because in watch task, if exist error in ts files the task fail and I need rerun de watch task. I add the force option and not fail watch task when exist error in ts files.

### CODE REVIEW

**Comment by joskuijpers** [comment]
+1 I want this

**Comment by ekosystem** [comment]
+1

**Comment by torpedro** [comment]
+1

**Comment by gronke** [comment]
:+1:

File: Gruntfile.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ module.exports = function(grunt) {
     tslint: {
       errors: {
         options: {
-          configuration: grunt.file.readJSON("tslint.json")
+          configuration: grunt.file.readJSON("tslint.json"),
+		  force:true
```
Comment (adidahiya) [inline Gruntfile.js]: use spaces, not tabs

Comment (adidahiya) [inline Gruntfile.js]: add a space after the colon

```
force: true
```

File: README.md [hunk 2]
```diff
@@ -39,7 +39,8 @@ A JSON configuration object passed into tslint.
 	grunt.initConfig({
 	  tslint: {
 	    options: {
-	      configuration: grunt.file.readJSON("tslint.json")
+	      configuration: grunt.file.readJSON("tslint.json"),
+		  force:true //set force to true to report tslint errors but not fail the task
```
Comment (adidahiya) [inline README.md]: use spaces, not tabs, and move the comment to its own line

```
// if force is true, tslint will report errors but not fail the task
force: true
```

File: tasks/tslint.js [hunk 3]
```diff
@@ -21,10 +21,14 @@ module.exports = function(grunt) {
   var Linter = require("tslint");
 
   grunt.registerMultiTask("tslint", "A linter for TypeScript.", function() {
+
```
Comment (adidahiya) [inline tasks/tslint.js]: remove extra newline

File: tasks/tslint.js [hunk 4]
```diff
@@ -21,10 +21,14 @@ module.exports = function(grunt) {
...
 
   grunt.registerMultiTask("tslint", "A linter for TypeScript.", function() {
+	  
     var options = this.options({
       formatter: "prose",
-      outputFile: null
+      outputFile: null,
+	  force: false
     });
+
```
Comment (adidahiya) [inline tasks/tslint.js]: remove trailing whitespace

File: tasks/tslint.js [hunk 5]
```diff
@@ -21,10 +21,14 @@ module.exports = function(grunt) {
...
   grunt.registerMultiTask("tslint", "A linter for TypeScript.", function() {
+	  
     var options = this.options({
       formatter: "prose",
-      outputFile: null
+      outputFile: null,
+	  force: false
     });
+	  
+	var force = options.force;
```
Comment (adidahiya) [inline tasks/tslint.js]: use spaces, not tabs

**Comment by adidahiya** [comment]
I'm happy to add the `force` option. This PR needs some revisions + resolved merge conflicts.

**Comment by sclausen** [comment]
Oh, I forgot: I need this really nice feature too :smile:

**Comment by mikemccaughan** [comment]
Any progress on this? I would love to see this incorporated. Is there anything I can do to help?

**Comment by JKillian** [comment]
It seems likely that this PR is stale/dead. Probably best to contribute a new PR for this feature if you'd like to see it!

**Comment by JKillian** [comment]
Closing this in favor of #50

**Comment by JKillian** [comment]
Just published grunt-tslint v3.2.0 which supports a `force` option

**Pull request closed without merging**